sonar-batch-protocol/src/main/gen-java/org/sonar/server/source/db/FileSourceDb.java

```diff
@@ -1527,12 +1552,9 @@ public boolean hasScmRevision() {
       public java.lang.String getScmRevision() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)